packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -206,18 +206,18 @@ describe('plugin-conversation', function () {
     describe('#list()', () => {
       let conversation1, conversation2;
 
-      before('create conversations', () => Promise.all([
+      before('create conversations', () =>
         webex.internal.conversation.create({
           displayName: 'test 1',
           participants
         })
-          .then((c) => { conversation1 = c; }),
-        webex.internal.conversation.create({
-          displayName: 'test 2',
-          participants
-        })
-          .then((c) => { conversation2 = c; })
-      ]));
+          .then((c) => { conversation1 = c; })
+          .then(() => webex.internal.conversation.create({
```

Comment:
Were there issues creating two spaces at the same time?

Reply:
Yes. Since they were in a promise.all they were coming out of order sometimes. This is problematic when I want to assert that a specific conversation comes first and then a second one comes in the next page of results